<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FZapr</class>
 <widget class="QWidget" name="FZapr">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список запросов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/Table.png</normaloff>:/res/Table.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Create.png</normaloff>:/res/Create.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Delete.png</normaloff>:/res/Delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Редактировать</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Modify.png</normaloff>:/res/Modify.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Refresh.png</normaloff>:/res/Refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnForm">
        <property name="text">
         <string>Сформировать</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/table_analysis.png</normaloff>:/res/1/table_analysis.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSpis">
        <property name="text">
         <string>Список</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/table_layout_grand_totals.png</normaloff>:/res/1/table_layout_grand_totals.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>697</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
